model/task_lifecycle.go

```diff
@@ -1472,7 +1498,7 @@ func UpdateVersionAndPatchStatusForBuilds(buildIds []string) error {
 			if p == nil {
 				return errors.Errorf("no patch found for version '%s'", buildVersion.Id)
 			}
-			if err = UpdatePatchStatus(p, newVersionStatus, bvMap[buildId]); err != nil {
+			if err = UpdatePatchStatus(p, newVersionStatus); err != nil {
```

Comment:
Is this supposed to happen after `checkUpdateBuildPRStatus`? I would have thought the patch would have to be updated first so that we can resolve the patch's latest status before we check the patch status in `checkUpdateBuildPRStatus`.

Reply:
With the latest fix to only update, we no longer check the patch status in  `checkUpdateBuildPRStatus ` so this is fine to happen after that function.